jupyterlab_bokeh_server/server.py

```diff
@@ -141,12 +144,177 @@ def cb():
 
     doc.add_periodic_callback(cb, 200)
 
+try:
```

Comment:
I recommend that we throw all of the gpu code into a separate file, and then restrict the try-except to importing that file.  That will also keep the cpu-only code clean of GPU details.